content-src/asrouter/schemas/message-format.md

```diff
@@ -82,104 +82,4 @@ If a tag that is not on the allowed is used, the text content will be extracted
 Grouping multiple allowed elements is not possible, only the first level will be used: `<u><b>text</b></u>` will be interpreted as `<u>text</u>`.
 
 ### Targeting attributes
-For a more in-depth explanation of JEXL syntax you can read the [Normady project docs](https://normandy.readthedocs.io/en/stable/user/filters.html#jexl-basics).
-
-Currently we expose the following targeting attributes that can be used by messages:
-
-Name | Type | Example value | Description
----  | ---  | ---           | ---
-`addonsInfo` | `Object` | [example below](#addonsinfo-example) | Information about the addons the user has installed
-`devToolsOpenedCount` | `Integer` | Number of usages of the web console or scratchpad
-`usesFirefoxSync` | `Boolean` | `true` | Does the user have a firefox account
-`isDefaultBrowser` | `Boolean` or `null` | Is Firefox the user's default browser? If we could not determine the default browser, this value is `null`
-`profileAgeCreated` | Number | `1522843725924` | Profile creation timestamp
-`profileAgeReset` | `Number` or `undefined` | `1522843725924` | When (if) the profile was reset
-`currentDate` | `Date` | `Date 2018-08-22T15:48:04.100Z` | Date object of current time in UTC
-`searchEngines` | `Object` | [example below](#searchengines-example) | Information about the current and available search engines
-`browserSettings.attribution` | `Object` or `undefined` | [example below](#attribution-example) | Attribution for the source of of where the browser was downloaded.
-`providerCohorts` | `Object` | `{onboarding: "hello"}` | Cohorts defined for all providers
-`previousSessionEnd` | `Number` | `1536325802800` | Timestamp in milliseconds of previously closed session
```

Comment:
this was lost in the move. It's implemented but not in Targeting but in ASRouter and passed down. Might be tricky to include in the regex tests.

Reply:
Ah! good catch